interpreter/misc_test.go

```diff
@@ -39,316 +39,30 @@ import (
 	"github.com/onflow/cadence/runtime"
 	"github.com/onflow/cadence/sema"
 	"github.com/onflow/cadence/stdlib"
+	"github.com/onflow/cadence/test_utils"
 	. "github.com/onflow/cadence/test_utils/common_utils"
 	. "github.com/onflow/cadence/test_utils/interpreter_utils"
 	. "github.com/onflow/cadence/test_utils/runtime_utils"
 	. "github.com/onflow/cadence/test_utils/sema_utils"
 )
 
-type ParseCheckAndInterpretOptions struct {
```

Comment:
These functions were replicated to `github.com/onflow/cadence/test_utils` with a hope to remove it from here, in a previous PR. So remove the implementation form there, but keeps names as-is just to avoid having to refactor all the tests (by converting them to variables with the same name) .